test/libsolidity/SemanticTest.h

```diff
@@ -58,9 +58,8 @@ class SemanticTest: public SolidityExecutionFramework, public EVMVersionRestrict
 	/// Compiles and deploys currently held source.
 	/// Returns true if deployment was successful, false otherwise.
 	bool deploy(std::string const& _contractName, u256 const& _value, bytes const& _arguments, std::map<std::string, solidity::test::Address> const& _libraries = {});
-
 private:
-	std::string m_source;
+	SourceMap m_source;
```

Comment:
For consistency with the reader class:
```suggestion
	SourceMap m_sources;
```